app/models/gemnasium_client.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'rest-client'
+require 'timeout'
+
+class GemnasiumClient
+  def initialize
+    api_key = Settings.GEMNASIUM_API_TOKEN
+    @api = 'https://X:' + api_key + '@api.gemnasium.com/v1'
```

Comment:
Can we set `@api` up with an attr_reader? That way, when the api is invoked in other methods in the class, we wouldn't need to use the `@`. For an example of the pattern, see https://github.com/sul-dlss/quimby/blob/master/app/models/puppetdb_client.rb#L6-L15